src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
+
+    private T[] myList;
+    private int sizeList;
+
+    public ArrayList() {
+        myList = (T[]) new Object[CAPACITY];
+        sizeList = 0;
+    }
 
     @Override
     public void add(T value) {
-
+        checkResizeList();
+        myList[sizeList] = value;
+        sizeList += 1;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index < 0) {
```

Comment:
what if my size = 1 and i want to put an element to 20 index?

Reply:
done